python/lsst/ts/tunablelaser/hardware.py

```diff
@@ -54,8 +55,8 @@ class CPU8000:
 
     Parameters
     ----------
-    commander : `TCPIPClient`
-        A reference to the tcp/ip client.
+    component : `Laser`
```

Comment:
This comment doesn't apply to this line. Please rename the file to something more descriptive than `hardware.py`, for instance `cpu8000.py` though that still doesn't mean much. Maybe include the name of the manufacturer? Also update the class description to something more meaningful and describe the difference with the MCPU800 class. Please also update the class description of the MCPU800 class.